repos/system_upgrade/el7toel8/models/systemfacts.py

```diff
@@ -60,12 +60,10 @@ class ActiveKernelModule(Model):
 class SELinux(Model):
     topic = SystemInfoTopic
 
-    # FIXME: fixme properly regarding the issue:
-    # # https://github.com/oamg/leapp-repository/issues/20
     runtime_mode = fields.Nullable(fields.StringEnum(['enforcing', 'permissive']))
     static_mode = fields.StringEnum(['enforcing', 'permissive', 'disabled'])
     enabled = fields.Boolean()
-    policy = fields.String()
+    policy = fields.StringEnum(['targeted', 'minimum', 'mls'])
```

Comment:
This field needs to be Nullable when selinux is disabled how it looks like

Reply:
Hmm...when selinux is disabled, still there would be set policy in the configuration file. Maybe instead of that, I would suggest use default policy (`targeted`), in case we cannot get it even from the config file. Not 100% about behaviour of the selinux function in such case.

Reply:
I mean, when selinux is disabled, it doesn't not mean it  is permanent state.

Reply:
Right, these `'targeted', 'minimum', 'mls'` are the only values we should receive in whatever state SELinux is in.

Reply:
see https://github.com/oamg/leapp-repository/issues/39

Reply:
Yes, that's purpose of this PR, to resolve #39  :) Btw. I checked functionality of `selinux.selinux_getpolicytype()`. So here are various results based on various invalid values in config file:
```python
# SELINUXTYPE=foo
>>> selinux.selinux_getpolicytype()
[0, 'foo']

# SELINUXTYPE=
>>> selinux.selinux_getpolicytype()
[0, '']

#
>>> selinux.selinux_getpolicytype()
[0, 'targeted']
```

Which is expected behaviour. Just in case of empty string, I would go with `targeted` anyway. In case someone will use custom policy, upgrade will not be enabled and we will now that we have to take care of it anyway. (in the meaning, that the value is probably important only in case selinux is enabled.. )